definitions/npm/create-react-class_v15.x.x/flow_v0.43.x/create-react-class_v15.x.x.js

```diff
@@ -0,0 +1,3 @@
+declare module 'create-react-class' {
+  declare module.exports: React$CreateClass;
```

Comment:
I think you want `declare export default` here. `declare var exports` also works.